include/zephyr/drivers/fuel_gauge.h

```diff
@@ -59,9 +59,11 @@ extern "C" {
 /** Retrieve word from SBS1.1 ManufactuerAccess */
 #define FUEL_GAUGE_SBS_MFR_ACCESS	    FUEL_GAUGE_RUNTIME_TO_FULL + 1
 /** Absolute state of charge (percent, 0-100) - expressed as % of design capacity */
-#define FUEL_GAUGE_STATE_OF_CHARGE	    FUEL_GAUGE_SBS_MFR_ACCESS + 1
+#define FUEL_GAUGE_ABSOLUTE_STATE_OF_CHARGE	    FUEL_GAUGE_SBS_MFR_ACCESS + 1
```

Comment:
Let's also rename/add to the get_property union below. I'd rather not overload the `state_of_charge` field with both asoc and rsoc.

Reply:
Oh man, that was an oversight - yikes - fixing now :-)

Reply:
@aaronemassey - ready for re-review, but for some reason, github won't let me request.